pkg/iptables/iptables.go

```diff
@@ -28,6 +30,12 @@ import (
 	"k8s.io/klog"
 )
 
+const (
+	IptablesCmd = "iptables"
+	VersionCmd  = "--version"
+	NftSubstr   = "nf_tables"
```

Comment:
These don't need to be publicly scoped. That said they're only used in one place and thus don't need to be constants.